bindings/tbdex_uniffi/src/http_client/exchanges.rs

```diff
@@ -76,8 +76,15 @@ pub fn get_exchange(
     Ok(Exchange::from_inner(inner_exchange))
 }
 
-pub fn get_exchange_ids(pfi_did_uri: String, bearer_did: Arc<BearerDid>) -> Result<Vec<String>> {
-    let exchange_ids =
-        tbdex::http_client::exchanges::get_exchange_ids(&pfi_did_uri, &bearer_did.0.clone())?;
+pub fn get_exchange_ids(
+    pfi_did_uri: String,
+    bearer_did: Arc<BearerDid>,
+    query_params: Option<GetExchangeIdsQueryParams>,
```

Comment:
if the query_param is absent, does it return the full list?

Reply:
that would be logic implemented on the PFI-side, but yes if not included then the expectation would be the full set of exchange IDs

perhaps we should add that language to [the spec](https://github.com/TBD54566975/tbdex/tree/main/specs/http-api#pagination), or perhaps we should have a maximum as the default and include pagination on the response (@jiyoontbd @mistermoe)

Reply:
if there's a million exchange between Alice and PFI already, it might be just a tad bit difficult for PFI to return all that in one response

so i agree with the idea of having a max. we could have it paginated by default showing the last X exchange... i don't really know what X should be. 50? 100?

Reply:
this is just a list of ids tho, right, and not the full list of messages (which is an Exchange)?